Dockerfile

```diff
@@ -12,11 +11,23 @@ ENV PATH $GRAILS_HOME/bin:$PATH
 RUN apt-get update && apt-get install -y --no-install-recommends software-properties-common \
     && add-apt-repository ppa:openjdk-r/ppa \
     && apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
```

Comment:
Not sure why build-essential, libudunits2-dev and python-dev are here?

Also, `git-core` is an old metapackage which just points to `git`, so just use that directly.